src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -228,17 +219,17 @@ private IServiceCollection BuildHostingServices()
             return services;
         }
 
-        private string ResolveApplicationBasePath(string applicationBasePath, string basePath)
+        private string ResolveContentRootPath(string contentRootPath, string basePath)
         {
-            if (string.IsNullOrEmpty(applicationBasePath))
+            if (string.IsNullOrEmpty(contentRootPath))
```

Comment:
This only works in IIS because @moozzyk's publish-iis will specify the contentRootPath as an environment variable in the web.config, correct? Otherwise basePath will be pointing to wwwroot by default, right?